orchestrator/taste-orchestrator.py

```diff
@@ -2128,6 +2183,24 @@ def DetectGUIsubSystems(AdaIncludePath):
     return guiSubsystems, AdaIncludePath
 
 
+def DetectROSBridgesubSystems(AdaIncludePath):
+    '''Detects the ROSBridge systems that will be built'''
+    g_stageLog.info("Detecting ROSBridge subSystems")
+    rosBridgeSubsystems = []
+    for line in os.popen("/bin/ls */*ros_bridge_code.cpp 2>/dev/null", 'r').readlines():
+        line = line.strip()
+        baseDir = re.sub(r'/.*', '', line)
```

Comment:
`line=line.strip()` can be inlined if it doesn't harm readability.

Reply:
Yup should be ok